app/components/subcases/side-nav/publication-subcase.hbs

```diff
@@ -1,4 +1,4 @@
-<li class="auk-sidebar__item auk-sidebar__item--styled auk-sidebar__item--inside-link {{if @selected 'auk-sidebar__item--active'}}">
+<li class="auk-sidebar__item auk-sidebar__item--styled">
```

Comment:
@brenner-company Not sure if this PR caused it, but the publication tab doesn't show the selected status anymore (narrow blue line at left side) 

Use https://kaleidos-dev.vlaanderen.be/dossiers/66A21DD8238A984711D56DE3/publicatie/671121B07F5A58BC608CBB6B to test

This branch:
![image](https://github.com/user-attachments/assets/84ab6734-915c-4136-87e8-b3ddca21b5ce)

On dev:
![image](https://github.com/user-attachments/assets/01036c29-2e48-45d8-8db8-9a233fe1ed88)

Also noticed that the styling is different for this tab as well (grey background instead of white for subcases)
![image](https://github.com/user-attachments/assets/f5e78f61-f255-4019-8b49-6b753f311636)

When fixing this, note that there can be more than 1 publication tab for 1 dossier. In other words, you can't simply check the route to enable the "active" style here, as there might be 2 or more publication tabs.

Reply:
@tdn Just to check everything off, did you do a `npm i` before previewing the changes (locally)?

Reply:
@brenner-company indeed, that fixed it. The background is still grey, but it's like that on DEV as well, so fine to merge